lib/KTable/index.vue

```diff
@@ -0,0 +1,505 @@
+
+          default:
+            return;
+        }
+
+        this.focusCell(nextRowIndex, nextColIndex);
+        this.focusedRowIndex = nextRowIndex === -1 ? null : nextRowIndex;
+        this.focusedColIndex = nextColIndex;
+
+        this.highlightHeader(nextColIndex);
+
+        event.preventDefault();
+      },
+      getCell(rowIndex, colIndex) {
+        if (rowIndex === -1) {
+          return this.$el.querySelector(`thead th:nth-child(${colIndex + 1})`);
+        } else {
+          return this.$el.querySelector(
+            `tbody tr:nth-child(${rowIndex + 1}) td:nth-child(${colIndex + 1})`
+          );
+        }
+      },
+      scrollCellIntoView(cell) {
+        if (cell) {
+          cell.scrollIntoView({ behavior: 'smooth', block: 'nearest', inline: 'nearest' });
+
+          // Adjust scroll position to account for sticky headers
+          const stickyHeader = this.$el.querySelector('.sticky-header');
+          const stickyColumn = this.$el.querySelector('.sticky-column');
+          const tableWrapper = this.$refs.tableWrapper;
+
+          const stickyHeaderHeight = stickyHeader ? stickyHeader.offsetHeight : 0;
+          const stickyColumnWidth = stickyColumn ? stickyColumn.offsetWidth : 0;
+
+          const cellRect = cell.getBoundingClientRect();
+          const wrapperRect = tableWrapper.getBoundingClientRect();
+
+          if (cellRect.top < wrapperRect.top + stickyHeaderHeight) {
+            tableWrapper.scrollTop -= wrapperRect.top + stickyHeaderHeight - cellRect.top;
+          }
+          if (cellRect.left < wrapperRect.left + stickyColumnWidth) {
+            tableWrapper.scrollLeft -= wrapperRect.left + stickyColumnWidth - cellRect.left;
+          }
+        }
+      },
+      focusCell(rowIndex, colIndex) {
+        let nextCell;
+        if (rowIndex === -1) {
+          nextCell = this.$el.querySelector(`thead th:nth-child(${colIndex + 1})`);
+        } else {
+          nextCell = this.$el.querySelector(
+            `tbody tr:nth-child(${rowIndex + 1}) td:nth-child(${colIndex + 1})`
+          );
+        }
+        // Ensured the focused cell is smoothly scrolled into view.
+        if (nextCell) {
+          nextCell.focus();
+          this.scrollCellIntoView(nextCell);
+        }
+      },
+
+      handleRowMouseOver(rowIndex) {
+        this.hoveredRowIndex = rowIndex;
+      },
+      handleRowMouseLeave() {
+        this.hoveredRowIndex = null;
+      },
+      setHighlightHeader(header, highlight) {
+        header.style.backgroundColor = highlight
+          ? this.$themePalette.grey.v_50
+          : this.$themePalette.white;
+      },
+      highlightHeader(colIndex) {
+        const headers = this.$el.querySelectorAll('thead th');
+        headers.forEach((header, index) => {
+          this.setHighlightHeader(header, index === colIndex);
+        });
+      },
+    },
+  };
+
+</script>
+
+
+<style scoped>
+.k-table-wrapper {
+  overflow: auto;
+  position: relative;
+  height:300px;
```

Comment:
Setting a minimum height or adjusting the height to auto could help to remove this side scroll bar and enhance the table's responsiveness in Kolibri.
![ktablewithscrollbar](https://github.com/user-attachments/assets/ce1f9dee-bf51-41f8-a22d-f33a54de5e28)

Reply:
I have added a prop to the table- `height` - that can be used to set the height of the table. It is set to `auto` by default for now.

Reply:
@BabyElias Do we need a new prop? I would assume that <KTable :style="{ height: ... }" /> should work if we remove the hardcoded value from inside that @LianaHarris360 pointed out? I think you could even set it to `'auto'` on line 469 to provide default behavior. It should still be properly overriden by `style`.

Reply:
Didn't check that one, ideally it should work. Looking into it.

Reply:
It works! Reverted this change and set default height of table as 'auto'